packages/editor-ui/src/components/HtmlEditor/HtmlEditor.vue

```diff
@@ -234,7 +238,6 @@ onMounted(() => {
 });
 
 onBeforeUnmount(() => {
-	if (isDirty.value) emit('update:model-value', readEditorValue());
```

Comment:
Why did you remove this? IIRC this is to catch the last changes that are potentially being debounced when the component unmounts

Reply:
I deleted it because it prevented use of defaultValue on the field, it would always reset it to empty. But that doesn't seem to be the case anymore 🤔 , so reverting it back